src/components/forms/Dropdown/index.ts

```diff
@@ -18,6 +18,7 @@ declare var PCore: any;
 class Dropdown extends FormComponentBase {
 
   @property( {attribute: false, type: Array} ) options;
+  @property( {attribute: true, type: Array} ) datasource = [];
```

Comment:
you are passing string value to the prop I guess

Reply:
Yes, thanks for noticing this, will update.